src/main/java/net/helix/pendulum/conf/BasePendulumConfig.java

```diff
@@ -804,7 +800,7 @@ protected void setBelowMaxDepthTransactionLimit(int maxAnalyzedTransactions) {
     @Override
     public int getStartRoundDelay() {return startRoundDelay; }
     @JsonProperty
-    @Parameter(names = {"--start-validator"}, description = ValidatorManagerConfig.Descriptions.START_ROUND_DELAY)
+    @Parameter(names = {"--start-validatorPath"}, description = ValidatorManagerConfig.Descriptions.START_ROUND_DELAY)
```

Comment:
parameter flags should follow: --a-b-c format. ie `--start-validatorPath` should be `--start-validator-path`. 
Any reason we need the "start" prefix?

Reply:
I have made a new commit, --start-validator prameter should remain unchanged.